docs/client-libraries-producers.md

```diff
@@ -41,33 +52,45 @@ Producer<String> producer = pulsarClient.newProducer(Schema.STRING)
 </Tabs>
 ````
 
-## Send messages
+## Publish messages
+
+This example shows how to publish messages using producers. The publish operation will be done until the broker tells us that the message has been successfully published. It will return the message id after the message is published successfully.
```

Comment:
Suggest moving this introductory paragraph next to the code example.

Reply:
+1